cedar-policy-validator/src/schema_file_format.rs

```diff
@@ -474,10 +482,8 @@ pub enum SchemaTypeVariant {
         element: Box<SchemaType>,
     },
     Record {
-        #[serde(with = "serde_with::rust::maps_duplicate_key_is_error")]
         attributes: BTreeMap<SmolStr, TypeOfAttribute>,
```

Comment:
does this need to be `AttributesTypeMap` instead of the `BTreeMap`?

Reply:
It could, but I think I prefer leaving that struct as just an implementation detail for the custom deserializer. I'd rather not have the struct at all, so this makes it possible to delete it later if I figure out has serde_with works without messing with any other code.